### ISSUE TITLE
Google LDAP: Radius authorize fails - Use of uninitialized value $radius_return in numeric eq

### ISSUE THREAD

**Issue by nathanbloemendaal78** [inverse-inc/packetfence#7528]
I have a PF 12.1 setup with a Google LDAP auth source. User lookup works, but authorize fails:

Feb 10 15:50:40 lis-dc-egd-01 httpd.aaa-docker-wrapper[2827]: httpd.aaa(7) INFO: [mac:xx:xx:xx:xx:xx:xx] handling radius autz request: from switch_ip => (x.x.x.x), connection_type => Wireless-802.11-EAP,switch_mac => (58:59:c2:85:31:14), mac => [xx:xx:xx:xx:xx:xx], port => 0, username => "test@school.com", ssid => eduroam (pf::radius::authorize)
Feb 10 15:50:40 lis-dc-egd-01 httpd.aaa-docker-wrapper[2827]: httpd.aaa(7) INFO: [mac:xx:xx:xx:xx:xx:xx] Instantiate profile xxx-802.1x-wireless (pf::Connection::ProfileFactory::_from_profile)
Feb 10 15:50:40 lis-dc-egd-01 httpd.aaa-docker-wrapper[2827]: httpd.aaa(7) INFO: [mac:xx:xx:xx:xx:xx:xx] Found authentication source(s) : 'xxx-google-ldap-new,local' for realm 'school.com' (pf::config::util::filter_authentication_sources)
Feb 10 15:50:40 lis-dc-egd-01 httpd.aaa-docker-wrapper[2827]: httpd.aaa(7) INFO: [mac:xx:xx:xx:xx:xx:xx] Using sources xx-google-ldap-new, local for matching (pf::authentication::match2)
Feb 10 15:50:40 lis-dc-egd-01 httpd.aaa-docker-wrapper[2827]: httpd.aaa(7) WARN: [mac:xx:xx:xx:xx:xx:xx] [xxx-google-ldap-new test-catchall] Searching for (&(|(mail=test@school.com))), from dc=teylingen-college,dc=nl, with scope sub (pf::Authentication::Source::LDAPSource::match_in_subclass)
Feb 10 15:50:40 lis-dc-egd-01 httpd.aaa-docker-wrapper[2827]: httpd.aaa(7) ERROR: [mac:xx:xx:xx:xx:xx:xx] radius authorize failed with error: SSL_cert_file /usr/local/pf/raddb/certs/google-ldap.crt can't be used: No such file or directory at /usr/local/pf/lib_perl/lib/perl5/IO/Socket/SSL.pm line 2375.
Feb 10 15:50:40 lis-dc-egd-01 httpd.aaa-docker-wrapper[2827]: httpd.aaa(7) WARN: [mac:xx:xx:xx:xx:xx:xx] Use of uninitialized value $radius_return in numeric eq (==) at /usr/local/pf/lib/pf/radius/rest.pm line 48.
Feb 10 15:50:40 lis-dc-egd-01 httpd.aaa-docker-wrapper[2827]: httpd.aaa(7) WARN: [mac:xx:xx:xx:xx:xx:xx] Use of uninitialized value $radius_return in numeric eq (==) at /usr/local/pf/lib/pf/radius/rest.pm line 53.
Feb 10 15:50:40 lis-dc-egd-01 httpd.aaa-docker-wrapper[2827]: httpd.aaa(7) WARN: [mac:xx:xx:xx:xx:xx:xx] Use of uninitialized value $radius_return in numeric eq (==) at /usr/local/pf/lib/pf/radius/rest.pm line 53.
Feb 10 15:50:40 lis-dc-egd-01 httpd.aaa-docker-wrapper[2827]: httpd.aaa(7) WARN: [mac:xx:xx:xx:xx:xx:xx] Use of uninitialized value $radius_return in numeric eq (==) at /usr/local/pf/lib/pf/radius/rest.pm line 53.

This error seems to have been fixed a long time ago in issue 5018 and 5020. I have patched this installation to the latest (12.1.0+20230206204220+769699847+0011+maintenance~12~1+bullseye1) over (12.1.0+20230126143508+759335432+0011+maintenance~12~1+bullseye1))

I have checked and double checked the config, but it is really straightforward, nothing fancy.
Did I miss something here, or did I hit a bug? Of course, I can provide additional logs and config files.

**Comment by jrouzierinverse** [comment]
Is the certificate file for google on the file system of PacketFence?
```
Feb 10 15:50:40 lis-dc-egd-01 httpd.aaa-docker-wrapper[2827]: httpd.aaa(7) ERROR: [mac:xx:xx:xx:xx:xx:xx] radius authorize failed with error: SSL_cert_file /usr/local/pf/raddb/certs/google-ldap.crt can't be used: No such file or directory at /usr/local/pf/lib_perl/lib/perl5/IO/Socket/SSL.pm line 2375.
```

**Comment by nathanbloemendaal78** [comment]
Yes it is, together with the key.

**Comment by nathanbloemendaal78** [comment]
No other suggestions? I rolled out a clean and patched 11.2 installation (the 12.1 was fresh and clean as well) with the same configuration, no errors and succesful authentication.
